.github/workflows/haskell.yml

```diff
@@ -142,7 +142,7 @@ jobs:
         TMP: ${{ runner.temp }}
         KEEP_WORKSPACE: 1
         CREATE_GOLDEN_FILES: 1
-      run: cabal test all --enable-tests --test-show-details=direct
+      run: cabal test all --enable-tests --test-show-details=direct --ghc-options="-threaded -rtsopts \"-with-rtsopts=-N1 -T\""
```

Comment:
Can you add a comment explaining why we can't use more than 1 thread here?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1062/commits/f115e7a4790770fbadeb3fcdf8a5d6962d32994d